compiler-cli/src/lsp.rs

```diff
@@ -390,28 +445,17 @@ fn error_to_diagnostic(error: &Error) -> Option<PublishDiagnosticsParams> {
             data: None,
         };
         let path = location.path.canonicalize().unwrap();
-        let mut file: String = "file://".into();
-        file.push_str(&path.as_os_str().to_string_lossy());
-        let uri = Url::parse(&file).unwrap();
-
-        let diagnostic_params = PublishDiagnosticsParams {
-            uri,
-            diagnostics: vec![diagnostic],
-            version: None,
-        };
 
-        Some(diagnostic_params)
+        Some((path, diagnostic))
     } else {
-        todo!("Locationless diagnostic for LSP")
+        None
```

Comment:
Could you make this a todo please? I need to see if there's a way to surface these other warnings.

Reply:
Sure. I missed that